### ISSUE TITLE
how to cancel the installation of apex? 

### ISSUE THREAD

**Issue by LouZhiX** [surbhardwaj/BERT-for-Question-Answering#2]
if your computer can't get online, your project couldn't work.
![image](https://user-images.githubusercontent.com/24652283/55047026-4c6faf80-507e-11e9-8937-25593bc67dd2.png)

**Comment by surbhardwaj** [comment]
Here apex is not installed. This error pops out from the modelling.py file of pytorch-pretrained-BERT code.
"try:
    from apex.normalization.fused_layer_norm import FusedLayerNorm as BertLayerNorm
except ImportError:
    logger.info("Better speed can be achieved with apex installed from https://www.github.com/nvidia/apex .")". 
Apex is used for parallel processing. My code uses pytorch-pretrained-BERT which is throwing this warning. 
Hope, it resolves your issue.

**Comment by LouZhiX** [comment]
Thanks
